docs/roles/system_manager/manage_users.md

```diff
@@ -64,6 +68,69 @@ Once you're certain that you're adding a new user, make sure that the following
 - Be particularly careful never to use them to log in to any user-accessible VMs (such as the SRDs)
 ```
 
+(adding_users_manually)=
+
+### {{woman}} {{man}} Modifying user SRE access
+
+Users will need to be added to the relevant security group before they can access an SRE. This will have been done by setting the `GroupName` in the `user_details_template.csv` (see {ref}`generate_user_csv`), but can me modified later by the following:
+
+- Log into the **SHM primary domain controller** (`DC1-SHM-<SHM ID>`) VM using the login credentials {ref}`stored in Azure Key Vault <roles_system_deployer_shm_remote_desktop>`
+- In Server Manager click `Tools > Active Directory Users and Computers`
+- Click on `Safe Haven Security Groups`
+- Find the group that the user needs to be added to. For each SRE there are the following groups:
+    - `SG <SRE ID> Data Administrators` (enables database privileges, not write access to `/data`)
+    - `SG <SRE ID> Research Users`
+    - `SG <SRE ID> System Administrators`
```

Comment:
This is good, is there an explanation in the documents for these groups and their meaning? That feels like a missing piece of the puzzle here.

It might be best to put those definitions in the documentation as a reference and link to it here (and in the section about creating users with the csv).

Reply:
It doesn't look like there's a good explanation for these groups anywhere. Do we think that based on the "Roles" it should be: 

- Researcher: `SG <SRE ID> Research Users`
- Investigator: `SG <SRE ID> Data Administrators`
- System Manager: `SG <SRE ID> System Administrators`

<img width="318" alt="Screenshot 2023-02-21 at 14 09 33" src="https://user-images.githubusercontent.com/5486164/220368191-6f8faf1b-da33-4f23-80dc-53b9f18490ea.png">

I'm unsure why for example researchers don't have database creation privileges? (assuming I'm correct you need to be in `SG <SRE ID> Data Administrators` for that

Reply:
I think all of those security groups apply to Researchers and it comes from principle of least privilege.

- Research Users - Probably most Researchers, not special permissions
- Data Administrators - Researchers who can create/modify/delete database tables. Given to a smaller number of Researchers to prevent accidents.
- System Administrators - Researchers with elevated privileges through `sudo`. Rarely used but could be useful in Tier0/1 to let groups manage their own packages.

@jemrobinson, @martintoreilly is that correct?

Reply:
The reason for normal users not to be able to create/delete arbitrary schemas is because some SREs have their input data in database form (and we don't want users to be able to change that). All users should be able to create tables in the `public` schema though.

Reply:
Ok, I will update this info - I think it should still go on the `System Manager -> Managing Data Safe Haven Users` page, but near the top and referenced later